<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="title_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1911</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;               
</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,15,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
       <item>
        <widget class="QLabel" name="portrait_Label">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>86</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="name_Label">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>姓名:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ID_Label">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>ID号:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1565</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="switchUser_Btn">
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>68</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
border: 2px solid #0D9DDB;
color: #0D9DDB;</string>
       </property>
       <property name="text">
        <string>切换用户</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>1921</width>
      <height>911</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #FFFFFF;
border:none;
color:#333333;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="7" column="0">
         <widget class="QPushButton" name="shutDown_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../release/source/image/leftButton/quit1.png</normaloff>
            <activeoff>../../release/source/image/leftButton/quit1.png</activeoff>../../release/source/image/leftButton/quit1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="ranking_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>排行</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/ranging.png</normaloff>
            <activeoff>:/new/prefix1/release/source/image/leftButton/rangingSelected.png</activeoff>:/new/prefix1/release/source/image/leftButton/ranging.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="setting_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/setup.png</normaloff>
            <activeoff>:/new/prefix1/release/source/image/leftButton/setupSelected.png</activeoff>:/new/prefix1/release/source/image/leftButton/setup.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="history_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>历史</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/history.png</normaloff>
            <activeoff>:/new/prefix1/release/source/image/leftButton/historySelected.png</activeoff>:/new/prefix1/release/source/image/leftButton/history.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="training_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>118</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>训练</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/training.png</normaloff>
            <normalon>:/new/prefix1/release/source/image/leftButton/trainingSelected.png</normalon>
            <disabledoff>:/new/prefix1/release/source/image/leftButton/trainingSelected.png</disabledoff>
            <disabledon>:/new/prefix1/release/source/image/leftButton/setup.png</disabledon>
            <activeoff>:/new/prefix1/release/source/image/leftButton/trainingSelected.png</activeoff>
            <selectedon>:/new/prefix1/release/source/image/leftButton/trainingSelected.png</selectedon>:/new/prefix1/release/source/image/leftButton/training.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="teach_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>处方</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/recipe.png</normaloff>
            <activeoff>:/new/prefix1/release/source/image/leftButton/recipeSelected.png</activeoff>:/new/prefix1/release/source/image/leftButton/recipe.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="quickStart_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>快速启动</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/quickIn.png</normaloff>
            <activeoff>:/new/prefix1/release/source/image/leftButton/quickInSelected.png</activeoff>
            <selectedon>:/new/prefix1/release/source/image/leftButton/quickInSelected.png</selectedon>:/new/prefix1/release/source/image/leftButton/quickIn.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="scope_Btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>范围</string>
          </property>
          <property name="icon">
           <iconset resource="../../image.qrc">
            <normaloff>:/new/prefix1/release/source/image/leftButton/scope1.png</normaloff>
            <activeoff>:/new/prefix1/release/source/image/leftButton/scopeSelected.png</activeoff>:/new/prefix1/release/source/image/leftButton/scope1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #F1F5F9;</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="train_page">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="TrainingManager" name="train_Widget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="scorp_page">
        <widget class="ScopeManager" name="scope_Widget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>1681</width>
           <height>881</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="TeachManager" name="teach_page"/>
       <widget class="QWidget" name="history_page">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="TrainingHistory" name="history_Widget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ranking_page">
        <widget class="RankingList" name="ranking_Widget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1700</width>
           <height>910</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="SettingDialog" name="setting_page"/>
       <widget class="QWidget" name="user_page"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScopeManager</class>
   <extends>QWidget</extends>
   <header>scopemanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TeachManager</class>
   <extends>QWidget</extends>
   <header>teachmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingHistory</class>
   <extends>QWidget</extends>
   <header>traininghistory.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RankingList</class>
   <extends>QWidget</extends>
   <header>rankinglist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrainingManager</class>
   <extends>QWidget</extends>
   <header>trainingmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingDialog</class>
   <extends>QWidget</extends>
   <header>settingdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../image.qrc"/>
 </resources>
 <connections/>
</ui>
